Tests/test_PDB_parse_pdb_header.py

```diff
@@ -8,10 +8,9 @@
 
 
 import unittest
-import warnings
 
 try:
-    import numpy
+    import numpy  # noqa F401
```

Comment:
I think adding the line ``del numpy`` also avoids the linting error, but either is fine.